aws_lambda_powertools/shared/constants.py

```diff
@@ -18,3 +18,5 @@
 
 XRAY_SDK_MODULE = "aws_xray_sdk"
 XRAY_SDK_CORE_MODULE = "aws_xray_sdk.core"
+
+EVENT_HANDLER_DEBUG_ENV: str = "POWERTOOLS_EVENT_HANDLER_DEBUG"
```

Comment:
no need for :str as it's a constant

Reply:
sorry it is probably my over zealous ide

Reply:
in this context `XRAY_SDK_CORE_MODULE` is not consistent :P

Reply:
haha I said it, then I went over to look... and I did the same in others except XRAY, then thought FML... that needs fixing